core/scss/components/card/_card.scss

```
@@ -0,0 +1,73 @@
+@charset "UTF-8";
+
+//
+// Card
+//
+// One of the primary building block compontents. This card component comes with
+// several fields to ensure flexibility.
+//
+// .su-card--horizontal - Postcard
+//
+// Markup: ../templates/components/card/card.twig
+//
+// Style guide: Components.Card
+//
+.su-card {
+  @include box-shadow('shallow');
+
+  > img {
+    // Hide the image on the smallest screen sizes.
+    @media #{$breakpoint-xs-only} {
+      display: none;
+    }
+  }
+}
+
+// Card contents spacing.
+.su-card__contents {
+  @include modular-margin;
+
+  // Ensure no bottom margins on last item.
+  *:last-child {
+    margin-bottom: 0;
+  }
+
+  // TODO: Refactor this to a mixin when typo headings available.
+  > h5 {
+    font-size: 13px;
+    font-weight: bold;
+    line-height: 17px;
+    margin-bottom: 11px;
+    margin-top: 0;
+    text-transform: uppercase;
+
+    @media #{$breakpoint-md} {
+      margin-bottom: 12px;
+    }
+
+    @media #{$breakpoint-xl} {
+      margin-bottom: 13px;
+    }
+
+  }
+
+  // TODO: Refactor this to a mixin when typo headings available.
+  > h3 {
+    font-size: 25px;
+    font-weight: bold;
+    line-height: 31px;
+    margin-bottom: 18px;
+    margin-top: 0;
+
+    @media #{$breakpoint-xl} {
+      margin-bottom: 19px;
```

See previous related comment on margin mixin. This could be:

```suggestion
    @include margin(null null 19px);
```